content/pages/community/operators-rules.md

```diff
@@ -0,0 +1,111 @@
+
+**Note**: The CoC only applies to XMPP Standards Foundation (XSF) members and venues. It does not automatically apply to all members of all XMPP channels. Do not report misconduct in venues outside of the control of the XSF to this room or the XSF Conduct Team, unless an XSF member is involved.
+
+## On-Topicness
+
+The channel started out as a place for operators to work together to improve the general health of the XMPP network, encompassing topics such as interoperability, reliability and availability. It can provide valuable information when just reading passively, but also when actively asking.
+
+In order to keep the signal-to-noise ratio useful for people listening in on the channel passively, it is helpful to keep off-topic discussions to a minimum. To give you an idea about what people expect from the channel, here is a non-exhaustive list of topics which fit into the scope of the channel:
+
+* Announcements and raising awareness about problematic actors in the ecosystem affecting end-users or services negatively, preferably with corresponding evidence so that everyone is able to form their own opinion.
+* Discussion about tools for server operators (e.g. TLS/DNS checkers, configuration linters, configuration management systems (to some extent)).
+* Announcements of security advisories which concern XMPP server or client software (so that operators can pass this on to their users) or tools very commonly used with XMPP server deployments (e.g. anything you’d find in a typical linux server installation; no hard lines here).
+* Asking for contact information for a specific XMPP service in order to discuss an issue "off-list" (in private).
+* Discussion of approaches to instant messaging spam counter measures.
+* Discussion of effects of service policies on interoperability.
+* Baselines or standards for interoperability between servers (e.g. TLS ciphers, certificate authorities, port numbers, DNS setups etc.).
+
+### Edge cases
+
+Some topics sometimes fit in the scope. If unsure, you can ask the moderators privately for confirmation beforehand. In general, try to not overuse these categories.
+
+* Announcements or information relevant to services or tools which may be of interest for service operators (e.g. search.jabber.network or xmpp.net).
+
+  Here, it is important to strike the balance between advertisment and useful announcement. In general, security advisories related to such services or tools are always on-topic. If the service is paid for and the announcements aims at acquiring more customers, it may easily be off-topic. This is not an advertisment or business development channel.
+
+### Off-Topic
+
+Since some of the on-topic cases can quickly derail into off-topic when debates become too political or ad-hominem, here are some examples where the line toward off-topic has been crossed to make things clearer.
```

Comment:
I would try to have the two subsections:

#### Examples of on-topic discussions
#### Examples of off-topic discussions

Reply:
Reformatted the section. cc @mwild1, because you originally broke the formatting up in the way it was before; I now merged those sections again and hopefully made it clearer.